doc/rados/configuration/pool-pg-config-ref.rst

```diff
@@ -255,6 +255,15 @@ Ceph configuration file.
 :Type: 32-bit Integer
 :Default: ``45``
 
+``osd max pg per osd hard ratio``
+
+:Description: The ratio of number of PGs per OSD allowed by the cluster before
+              OSD refuses to create new PGs. OSD stops creates new PGs the number
+              of PGs it serves exceeds
+              ``osd max pg per osd hard ratio`` \* ``mon max pg per osd``.
```

Comment:
No offence. I can't understand the last sentence well:-)

Reply:
#18373 HTH.